auth/src/main/java/org/aerogear/mobile/security/SecurityService.java

```diff
@@ -16,70 +17,87 @@
  * Checks can be run individually using {@link #check(SecurityCheckType)} , or can be chained
  * together using an {@link SyncSecurityCheckExecutor} by using {@link #getCheckExecutor()}.
  */
-public class SecurityService implements ServiceModule{
+public class SecurityService implements ServiceModule {
     private final static String TYPE = "security";
 
+    /**
+     * {@link MobileCore}
+     */
```

Comment:
This javadoc is useless. If we want to document private members, then the comment must give some added value, otherwise it just enlarges the sources.